CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
-## 2.0.0
+# 3.0.0
+ - Added support to handle full arrays instead of only first element
+ - Changed default destination field to [source field]_translation in order to fix bug
```

Comment:
please add a link to an issue or some more information in other shape that explains the reasoning behind this